types/milesight/CT305/system.schema.json

```diff
@@ -0,0 +1,54 @@
+{
+  "$id": "https://akenza.io/milesight/CT305/system.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "processingType": "uplink_decoder",
+  "topic": "system",
```

Comment:
should this be lifecycle?

Reply:
This usually only gets sent after a boot. So its not really a lifecycle.